lib/CDVTestLocal.js

```diff
@@ -42,11 +31,6 @@ export default class CDVTestLocal extends CDVTest {
     if (appiumVersion) {
       logger.echo(`Appium installed, appium version: ${appiumVersion}`);
     }
-    // check appium is available
-    if (!this.isAppiumAvailable()) {
-      return new Error('Appium is not running or available, run appium &');
-    }
-    logger.ok('Checking appium is running & available');
     return null;
```

Comment:
@chauthai it's fine by me. Verifying appium is running is an extra. Just wanted the clarification on why it was removed.